caches/clan_test.go

```diff
@@ -0,0 +1,128 @@
+package caches_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	uuid "github.com/satori/go.uuid"
+	"github.com/topfreegames/khan/models"
+	"github.com/topfreegames/khan/testing"
+)
+
+var _ = Describe("Clan Cache", func() {
+	var testDb models.DB
+
+	BeforeEach(func() {
+		var err error
+		testDb, err = testing.GetTestDB()
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	Describe("Clans Summaries", func() {
+		It("Should return a cached payload for a second call made immediately after the first", func() {
+			gameID := uuid.NewV4().String()
+			_, clans, err := models.GetTestClans(testDb, gameID, "test-sort-clan", 10)
+			Expect(err).NotTo(HaveOccurred())
+
+			var publicIDs []string
+			clansMap := make(map[string]int)
+			for i, clan := range clans {
+				publicIDs = append(publicIDs, clan.PublicID)
+				clansMap[clan.PublicID] = i + 1
+			}
+
+			cache := testing.GetTestClansSummariesCache()
+
+			// first call
+			clansSummaries, err := cache.GetClansSummaries(testDb, gameID, publicIDs)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(clansSummaries)).To(Equal(len(clans)))
+			for _, clanPayload := range clansSummaries {
```

Comment:
(Reminder) Consider refactoring the "assertion" to a method, like we discussed in person.

Reply:
Done!